dtream1/main.cpp

```diff
@@ -1,8 +1,13 @@
 #include <iostream>
 #include <gmock/gmock.h>
+using namespace testing;
 
 class Cal {
 public:
+
+	int getSum(int a, int b) {
+		return a+b;
+
```

Comment:
머지 과정 중에 문제가 생긴 것 같습니다.
 ```suggestion
   	int getSum(int a, int b) {
		        return a+b;
	     }
```